app/pandas.ipynb

```diff
@@ -184,9 +181,7 @@
    "cell_type": "code",
    "execution_count": 16,
    "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Birt year\"] = datetime.now().year - df[\"Age\"]",
```

Comment:
There is a typo in the column name 'Birt year'. It should be 'Birth year'. This will cause issues when trying to access or manipulate this column later.